contracts/staking/contracts/src/fees/MixinExchangeFees.sol

```diff
@@ -232,7 +220,8 @@ contract MixinExchangeFees is
             ));
         }
         if (totalRewardsPaid > 0) {
-            _depositIntoStakingPoolRewardVault(totalRewardsPaid);
+            address payable rewardVaultAddress = address(uint160(address(rewardVault)));
```

Comment:
Are you doing this to cast `address(rewardVault)` to `address payable`?

Reply:
Yep, seems this is the only way =/

Reply:
Yeah I couldn't find a better way to do it, either.